src/System.IO/tests/TextReader/TextReaderTests.cs

```diff
@@ -70,6 +70,18 @@ public void TestRead()
             }
         }
 
+        [Fact]
+        public void ReadZeroCharcters()
+        {
+            if (!PlatformDetection.IsFullFramework)
```

Comment:
use static exclusion instead of runtime, ie
`[SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "#nnnn not fixed on .NET Framework")] `